CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -1179,7 +1179,7 @@ void __CFInitialize(void) {
     if (!__CFInitialized && !__CFInitializing) {
         __CFInitializing = 1;
 
-#if __HAS_DISPATCH__
+#if __HAS_DISPATCH__ && !TARGET_OS_MAC
```

Comment:
Ideally we would use the deployment version - but this is still forwards/backwards incompatible.  @parkera - any thoughts?  I'm tempted to say we merge this even with that limitation as this is blocking PRs for other platforms.

Reply:
This code is designed to go with the corresponding libdispatch, so if we are making a change to both (or a change to dispatch has already been made), they would go together into a Swift release.